modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -15,8 +15,18 @@ Specifically, it protects a cluster from large-scale infrastructure failure, thr
 Each group is created by an appropriately authorized administrator, and specified to contain a subset of the nodes within a Couchbase Cluster.
 Following group-definition and rebalance, the active vBuckets for any defined bucket are located on one group, while the corresponding replicas are located on another group: indexes and _index replicas_, maintained by the Index Service, are distributed across groups in a similar way.
 
-A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
+A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, then the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
+
+NOTE: For the vBuckets and replica indexes to be automatically promoted to active, the conditions specified in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must apply.
+
+See xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover].
 In 7.1+, this is achieved by setting the maximum count for auto-failover to a value equal to or greater than the number of nodes in the server group.
+
+Note that In 7.1+, _automatic failover_ can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering _auto-failover for server groups_.
+Consequently, in order to ensure successful auto-failover of a server group, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.
```

Comment:
1. The sentence in line 26 should be changed to **not** use the phrase "in order to ensure successful auto-failover" -- instead, use the phrase "for auto-failover of a server group to be possible":

Consequently, for auto-failover of a server group to be possible, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.  

2. And, the following additional info should be included for clarity and emphasis in the same paragraph:

Note that for auto-failover to occur, all of the auto-failover constraints listed in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must be met, including the majority quorum requirement -- that the remaining nodes must be able to form a majority quorum to be able to initiate an auto-failover.  If, for example, you have two server groups, with equal number of nodes in each, and if all of the nodes in one of the server groups fail at the same time, even if you have the maximum count for auto-failover set to a value equal to or greater than the number of nodes in the server group that failed, auto-failover cannot occur since the remaining nodes in the remaining server group cannot form a majority quorum.